### PR TITLE
fix #152766: Add keyboard shortcuts for up bow and down bow symbols

### PR DESCRIPTION
**Pull request by divya-urs**

### CODE REVIEW

File: mscore/shortcut.cpp [hunk 1]
```diff
@@ -604,6 +604,26 @@ Shortcut Shortcut::_sc[] = {
       {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY,
+         "add-stringsUpBow",
+         QT_TRANSLATE_NOOP("action","Up bow"),
+         QT_TRANSLATE_NOOP("action","Toggle Up bow"),
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `up`, lowercase

File: mscore/shortcut.cpp [hunk 2]
```diff
@@ -604,6 +604,26 @@ Shortcut Shortcut::_sc[] = {
...
+         0,
+         Icons::Invalid_ICON,
+         Qt::WindowShortcut
+         },
+      {
+         MsWidget::SCORE_TAB,
+         STATE_NORMAL | STATE_NOTE_ENTRY,
+         "add-stringsDownBow",
+         QT_TRANSLATE_NOOP("action","Down bow"),
+         QT_TRANSLATE_NOOP("action","Toggle Down bow"),
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `down`, lowercase

**Comment by divya-urs** [comment]
Is it fine now, or do I have to make more changes?

File: mscore/shortcut.cpp [hunk 3]
```diff
@@ -604,6 +604,26 @@ Shortcut Shortcut::_sc[] = {
...
+         QT_TRANSLATE_NOOP("action","Toggle Up bow"),
+         0,
+         Icons::Invalid_ICON,
+         Qt::WindowShortcut
+         },
+      {
+         MsWidget::SCORE_TAB,
+         STATE_NORMAL | STATE_NOTE_ENTRY,
+         "add-stringsDownBow",
+         QT_TRANSLATE_NOOP("action","Down bow"),
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `Bow!, uppercase here, (sorry, missed it first time round)

File: mscore/shortcut.cpp [hunk 4]
```diff
@@ -604,6 +604,26 @@ Shortcut Shortcut::_sc[] = {
       {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY,
+         "add-stringsUpBow",
+         QT_TRANSLATE_NOOP("action","Up bow"),
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `Bow`, uppercase here, (sorry, missed it first time round)

File: mscore/shortcut.cpp (hunk 3 above)
Comment (divya-urs) [inline mscore/shortcut.cpp]: So it should be "action", "Down Bow" ?

**Comment by divya-urs** [comment]
I'm having difficulty squashing into a single commit. I'm sorry, I'm new to using GitHub.

**Comment by lasconic** [comment]
What type of difficulty do you have?
There are 5 commits in this branch on top of master, see https://github.com/musescore/MuseScore/pull/2924/commits
We want to have a single one.

On the command line, in the MuseScore directory:
```
$ git checkout 152766-keyboard_shortcuts_bowing_symbols
$ git rebase -i HEAD~5
```
a text editor should open. For all the lines but the first one, replace the word "pick" by "squash" and save the file. Then a new text editor will open, remove all the lines but the one of the right commit message. Read http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html again.

**Comment by divya-urs** [comment]
I did this, but in the text editor I see more than 5 commits. The first commit is a8f4268.

**Comment by Jojo-Schmitz** [comment]
This doesn't look right at all

File: mscore/shortcut.cpp (hunk 3 above)
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: yes

**Comment by divya-urs** [comment]
I must have made some mistake. I'll close this pull request and open another one.

**Pull request closed without merging**